myapp/api/views/views.py

```diff
@@ -173,6 +173,64 @@ class OfficerViewSet(ReadOnlyModelViewSet):
     serializer_class = CustomUserSerializer
     permission_classes = [IsAuthenticated]
 
+class InducteeViewSet(ReadOnlyModelViewSet):
+    queryset_users = CustomUser.objects.filter(groups__name='inductee')
+    queryset_inductees = []
+    for user in queryset_users:
+        queryset_inductees.append(Inductee.objects.filter(user=user.user_id).first())
+        print("test")
+        print(queryset_inductees);
+    
+    serializer_class_user = CustomUserSerializer
+    serializer_class_inductee = InducteeSerializer
+    permission_classes = [IsAuthenticated]
```

Comment:
Maybe expand this so that this api is only accessible to admins. You can use [HasAdminPermissions] instead of [IsAuthenticated]

Reply:
Fixed officer, inductee, and outreach apis. May still need to be changed for other apis.